app/src/main/java/com/summer/internship/tvtracker/PopularFragment.kt

```diff
@@ -0,0 +1,16 @@
+package com.summer.internship.tvtracker
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.summer.internship.tvtracker.databinding.FragmentPopularBinding
+
+private lateinit var binding: FragmentPopularBinding
```

Comment:
keep the binding inside the PopularFragment class and also give me an explanation why you put it outside of it

Reply:
i didn't realise i put it outside ..sorry :)))

Reply:
that's why code review was made for